docs/migrations/00574.md

```diff
@@ -0,0 +1,9 @@
+## Toast
+
+#### Changed the default `closeable` value from `true` to `false`. Also changed the default notification presentation type to 'inline' that results in default styles change. To render a notification as a toast the default props needs to be overwritten.
+
+```diff
+-  <Toast>Message</Toast>
++  import {TYPE} from 'baseui/toast'
++  <Toast notificationType={TYPE.toast} closeable={true}>Message</Toast>
```

Comment:
can we do this in a non-breaking manner, by using defaults for the original behaviour?